packages/components/text-input/src/text-input.ts

```diff
@@ -19,33 +14,29 @@ let nextUniqueId = 0;
 /**
  * Single line text input component.
  *
+ * @csspart wrapper - The input's wrapper
  * @slot prefix - Content shown before the input
  * @slot input - The slot for the input element
  * @slot suffix - Content shown after the input
+ * @slot error-text - The error text to display
+ * @slot hint-text - The hint text to display
```

Comment:
There is no `hint-text` slot in the code?

Reply:
They are part of the `FormControlMixin`. Should I list them here, or in the mixin?

Reply:
In the mixin I guess.

Reply:
does it also show up in the documentation page when it's inherited from a mixin?

Reply:
are they really slots though? aren't they properties now? and depending on the value of the property/state you show the elements? because i see `<sl-error>` in the render function here